Feature/storybook (#40)
infinum/eightshift-frontend-libs: blocks/init/custom/accordion-item/docs/readme.md

# Accordion Item Block

> Block used to provide accordion item functionality to your project. This block is not designed to be used as a standa alone block, but rather to be used as a accordion block item inside parents inner block.

Reviewer: ```suggestion
Block used to provide accordion-item functionality to your project. This block is not designed to be used as a standalone block, but rather to be used as an accordion block item inside the parents' inner block.
```
